qulice-checkstyle/src/test/resources/com/qulice/checkstyle/Main.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Hello.
+ */
+package foo;
+/**
+ * Simple.
+ * @version $Id $
+ * @author John Smith (john@example.com)
+ */
+public final class Main {
```

Comment:
@prahladyeri same here, please rename the class to a more appropriate name, something that will tell me what is tested here